.github/workflows/update-generated-files.yaml

```diff
@@ -0,0 +1,32 @@
+name: Update generated files
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  update_genereated_files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Build
+        run: npm run build
+
+      - name: Create pull pull-requests
```

Comment:
I wanted to use a condition to run this step, but [there's currently a bug in action runners](https://github.com/actions/runner/issues/2033) that prevents 3rd party actions from running.